services/classic/management/http.go

```diff
@@ -111,7 +107,7 @@ func (client client) createHTTPClient() (*http.Client, error) {
 		Transport: &http.Transport{
 			Proxy: http.ProxyFromEnvironment,
 			TLSClientConfig: &tls.Config{
-				Renegotiation: tls.RenegotiateOnceAsClient,
+				Renegotiation: tls.RenegotiateFreelyAsClient,
```

Comment:
Curious about this change and if it's really necessary?  Perhaps it would be better to make it user-configurable, i.e. add it to `ClientConfig` and leave the default as `RenegotiateOnceAsClient`.

Reply:
This was an oversight. I'd reverted in my commit 092dc5b a short while back.